tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala

```diff
@@ -897,4 +897,67 @@ class CompletionWorkspaceSuite extends BaseCompletionSuite {
            |""".stripMargin
     ),
   )
+
+  checkEdit(
+    "directly-in-pkg".tag(IgnoreScalaVersion.forLessThan("3.2.2")),
+    """|package a:
+       |  object Y:
+       |    val bar = 123
+       |  val fooBar = 123
+       |
+       |package b:
+       |  def main() = fooB@@
+       |""".stripMargin,
+    """|import a.fooBar
+       |package a:
+       |  object Y:
+       |    val bar = 123
+       |  val fooBar = 123
+       |
+       |package b:
+       |  def main() = fooBar
+       |""".stripMargin,
+  )
+
+  check(
+    "nested-pkg".tag(IgnoreScalaVersion.forLessThan("3.2.2")),
+    """|package a:
+       |  package c: // some comment
+       |    def increment2 = 2
+       |  def increment = 1
+       |
+       |package d:
+       |  val increment3 = 3
+       |
+       |
+       |package b:
+       |  def main: Unit = incre@@
+       |""".stripMargin,
+    """|increment3: Int
```

Comment:
Any dea why this isn't ordered? Not important for this PR though

Reply:
`increment3` is a `val`, the other two are `def`s, I think this causes this ordering